integrations/dd-sdk-android-timber/api/dd-sdk-android-timber.api

```diff
@@ -1,4 +1,5 @@
 public final class com/datadog/android/timber/DatadogTree : timber/log/Timber$Tree {
+	public static final field TIMBER_TAG_ATTRIBUTE Ljava/lang/String;
```

Comment:
Is this relevant to this PR?

Reply:
no, but still needs to be updated, so it comes in a different commit